src/storage/src/upsert/types.rs

```diff
@@ -1089,6 +1091,58 @@ where
         Ok(())
     }
 
+    /// Insert or delete for all `puts` keys, prioritizing the last value for
+    /// repeated keys.
+    ///
+    /// Version of `multi_put` that uses the given stats, for when the caller
+    /// has a better way of correctly computing those stats.
```

Comment:
Can you explain what is the change related to stats and why they can't be computed in here?

Reply:
Updated the docstring to 

```
    /// Version of `multi_put` that uses the given stats, for when the caller
    /// cannot provide [`PutValues`](PutValue) with meaningful
    /// `previous_value_metadata`. This is the case then the caller is updating
    /// state without having previously read state, which latter would yield the
    /// required [`ValueMetadata`]. In those cases the caller hopefully has a
    /// better way of computing the same stats.
```